### PR TITLE
rename tutorial

### PR DESCRIPTION
**Pull request by sethladd**
Closes https://github.com/flutter/flutter/issues/3309

### CODE REVIEW

**Comment by sethladd** [comment]
cc @qchong

File: firebase.json [hunk 1]
```diff
@@ -11,6 +11,11 @@
   	  "source" : "/widgets/",
   	  "destination" : "/basic-widgets/",
   	  "type" : 301
-  	}
+  	},
+    {
+      "source" : "/tutorial/",
+      "destination" : "/introducing-flutters-widget-framework/",
```
Comment (Hixie) [inline firebase.json]: that's a bit of a mouthful if we're sending it around... Could we have a shorter URL?

Comment (Hixie) [inline firebase.json]: e.g. "widgets-intro"

File: introducing-flutters-widget-framework.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: Flutter Tutorial
-permalink: /tutorial/
+title: Introducing Flutter's Widget Framework
```
Comment (Hixie) [inline introducing-flutters-widget-framework.md]: I think I'd prefer "Introduction to..."

**Comment by Hixie** [comment]
LGTM

File: firebase.json (hunk 1 above)
Comment (sethladd) [inline firebase.json]: Trying to optimize for SEO, I'm assuming people are cutting and pasting, or searching and clicking, and not manually typing these paths.

Comment (Hixie) [inline firebase.json]: I wouldn't worry too much about SEO. If the page is useful, it'll rise in the rankings. The URL isn't going to make any noticeable difference to the rankings in practice.

Comment (sethladd) [inline firebase.json]: Renamed.

**Pull request merged**